gtfs-netex-test/configuration.py

```diff
@@ -20,7 +20,8 @@
 defaults["os"]= "windows"
 input_dir = "d:/input"
 processing_data =  "d:/aux_testing_processing"
-
+input_dir = "C:/Users/ue71603/MG_Daten/conversion/input"
```

Comment:
Nope, not absolute paths.

Reply:
I move this to local_config. those are my paths.